### ISSUE TITLE
pip install omsdk fails with an error message " Cannot uninstall 'PyYAML'. It is a distutlis insatlled project and thus we cannot accurately determine which files belong to it which would lead to only a partial uninstall. "

### ISSUE THREAD

**Issue by blrbhaskar** [dell/omsdk#4]
# QwikContext
pip install omsdk fails with an error message " Cannot uninstall 'PyYAML'. It is a distutlis insatlled project and thus we cannot accurately determine which files belong to it which would lead to only a partial uninstall."

![pip_install_omsdk_error](https://user-images.githubusercontent.com/32034374/38987208-98d47b3e-43ec-11e8-931d-3b48f4ad678c.JPG)

# Details
python version: 2.7.5
rhel version:7.3
pip version: 10.0.0

# Repro Steps
pip install omsdk

# Impact
omsdk installation fails from pypi 

# NeedBy
urgent

# Severity
medium

**Comment by saishivarcr** [comment]
Work-around:
pip install --ignore-installed pyyaml omsdk

**Comment by rajeevarakkal** [comment]
This issue is already documented in our readme section

Also few history here

https://github.com/pypa/pip/issues/5247

and RCA says

"See #4805#4805, pip 10 no longer uninstalls distutils packages  and #3389 for the history. Basically, pip cannot properly uninstall packages installed by "pure" distutils, because distutils doesn't record enough metadata for us to do so. We've previously removed the installation metadata, so that it looks like we did the install, but we've had to leave files behind. That causes issues. Since pip 8, we've been trying to stop doing this, because it's a cause of problems, but our initial attempt was reverted because it affected too many people at the time. With pip 10, we've finally stopped trying to uninstall distutils packages, and now we report the problem to the user as you see here.

Basically, if you (or some infrastructure you use) installed a package using distutils, you need to manage (and in particular) uninstall it "using distutils". Unfortunately distutils doesn't include an uninstall command, so "uninstall using distutils" means manually removing the package."
